### ISSUE TITLE
automate  import archive from GPX

### ISSUE THREAD

**Issue by noxqs** [vitalidze/traccar-web#1214]
Great work,

Currently I think I can only import GPX by clicking on the IMPORT button.
I would like to automate this at the end of every day.
How would you do that ? Is it already possible ? I can't find it in the rest and api doc.

**Comment by noxqs** [comment]
I think I understand a bit more; there is a POST to https://d.traccar.litvak.su/traccar/import/gpx?deviceId=[int]
I presume the gpx data is attached in the POST.

**Comment by vitalidze** [comment]
Yes, that's correct. You will need to be authorised in a similar way as you do to access the REST API.

Please note that the demo server should not be used for the production data. It is a subject to be wiped from time to time.
